spec/Factory/CsvSerializerFactorySpec.php

```diff
@@ -0,0 +1,19 @@
+<?php
+declare(strict_types=1);
+
+namespace spec\BitBag\SyliusWishlistPlugin\Factory;
+
+use BitBag\SyliusWishlistPlugin\Factory\CsvSerializerFactory;
+use BitBag\SyliusWishlistPlugin\Factory\CsvSerializerFactoryInterface;
+use PhpSpec\ObjectBehavior;
+
+final class CsvSerializerFactorySpec extends ObjectBehavior
```

Comment:
Missing spec scenarios. If there is no logic, I would remove the spec file. Please see all occurrences like this, as it's not the single case.